lib/anagrams.rb

```diff
@@ -0,0 +1,38 @@
+class Anagrams
+  def initialize(anagram_file)
+    @anagram_file = anagram_file
+  end
+
+  def group_anagrams
+    anagram_group = {}
+    read_file.group_by do |word|
+      sorted_word = word.chars.sort.join
```

Comment:
@zzokki81 I don't think this will work for anagrams such as `ABS` and `AB's`. You should ignore case and non-letter chars. Try adding some tests and you'll probably see them fail and not match words with special chars or upper/lower case letters.